src/guides/factory-reset.html

```diff
@@ -42,13 +42,23 @@
           </ul></span>
       </li>
       <li>Remove power from the system.</li>
-      <li>Perform these steps in close sequence:
+      <li>Power up the system:
         <ul>
-          <li>Reapply power and make sure the system has an Internet connection.</li>
-          <li>Within 5 seconds, press and hold the red button.</li>
-          <li>Keep the button pressed until the yellow LED starts flashing.</li><ul><li>The Home Assistant operating system now downloads the Supervisor and the Core.</li>
-            <li>Due to the downloads, it can take a while until Home Assistant becomes accessible again.</li>
-          </ul>
+          <li>Press and hold the red button.</li>
+          <li>Make sure the system has an Internet connection and reapply power.</li>
+          <li>Keep the button pressed until the yellow LED starts flashing.</li>
```

Comment:
Since https://github.com/home-assistant/operating-system/pull/3536 (13.1.rc1+) the LED will turn on and stay lit until the button is released. Before, when the signal was registered, it took couple of seconds for the Linux to start and the LED started to blink. So the universal way to describe it would be:

```suggestion
          <li>Keep the button pressed until the yellow LED turns on or starts flashing.</li>
```